src/MSAL.Frameworks.props

```diff
@@ -1,16 +1,16 @@
 <Project>
   <PropertyGroup>
-    <TargetFrameworkNetDesktop>net45</TargetFrameworkNetDesktop>
+    <TargetFrameworkNetDesktop>net461</TargetFrameworkNetDesktop>
```

Comment:
>net461 [](start = 31, length = 6)

JM?

Reply:
@jmprieur can you provide validation/signoff that moving to net461 for net desktop is acceptable for 3.x?